tuf-spec.md

```diff
@@ -1368,20 +1368,24 @@ it in the next step.
 
 3. **Check for a rollback attack.**
 
-  1. The version number of the trusted timestamp metadata file, if
-    any, MUST be less than the version number of the new timestamp
-    metadata file. If the new timestamp metadata version is less than the trusted
-    timestamp metadata version, discard it, abort the update cycle, and
-    report the potential rollback attack. In case they are equal, discard the new
-    timestamp metadata and abort the update cycle. This is normal and it
-    shouldn't raise any error. The reason for aborting the update process is that
-    there shouldn't be any changes in the content of this, or any other metadata
-    files too, considering it has the same version as the already trusted one.
-
-  2. The version number of the snapshot metadata file in the
-    trusted timestamp metadata file, if any, MUST be less than or equal to its
-    version number in the new timestamp metadata file. If not, discard the new
-    timestamp metadata file, abort the update cycle, and report the failure.
+  1. The version number of the trusted timestamp metadata file, if any, MUST be
+     less than the version number of the new timestamp metadata file. If the
+     new timestamp metadata version is less than the trusted timestamp metadata
+     version, discard it, abort the update cycle, and report the potential
+     rollback attack. In case they are equal, discard the new timestamp
+     metadata and abort the update cycle. This is normal and it shouldn't raise
+     any error. The reason for aborting the update process is that there
+     shouldn't be any changes in the content of this, or any other metadata
+     files too, considering it has the same version as the already trusted one.
+
+  2. The new timestamp metadata file MUST only contain the description of the
+     snapshot metadata file.  If not, discard the new snapshot metadata file,
+     abort the cycle, and report the failure.
```

Comment:
```suggestion
  2. The new timestamp metadata file MUST only contain the description of the
     snapshot metadata file.  If not, discard the new timestamp metadata file,
     abort the cycle, and report the failure.
```

Just to be clear, here we're checking that timestamp's `meta` lists _only_ snapshot?